src/styles/index.scss

```diff
@@ -1,3 +1,69 @@
-body {
+@import 'variables';
+
+html {
+  font-family: Arial, sans-serif;
+}
+
+* {
   margin: 0;
+  padding: 0;
+}
+
+body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+@each $day, $index in $days {
+  $index: $index + 6;
+  .calendar--start-day-#{$day} > :first-child {
+    margin-left: calc((#{$calendar-day-size} + #{$calendar-gap}) * #{$index});
+  }
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the SCSS code should not contain nested loops. Please, refactor the code to avoid nesting loops inside `@each` or `@for` structures.